indigo/tests/test_toc_builder_base.py

```diff
@@ -36,7 +36,7 @@ def test_toc_item_simple(self):
             document_xml=document_fixture(text="hi"),
             language=self.eng)
 
-        elem = doc.doc.root.xpath("//*[@id='section-1']")[0]
+        elem = doc.doc.root.xpath("//*[@eId='section-1']")[0]
```

Comment:
This document fixture should change so that it uses the correct id format. This (and probably a bunch of other tests) will then have to change to `'sec_1'`.